uint8 ID
uint32 timestamp
float32 PosX
float32 PosY
uint8 Blinker
uint8 CutState
float32 VrelX
uint8 MType
uint8 Status
uint8 Brake
uint8 Valid
float32 Length
float32 Width
uint8 Age
uint8 ObsLane
uint8 CIPV
float32 AngleRate
float32 ScaleChange
float32 ArelX
float32 Angle
uint8 Replaced
